docs/AppBar.md

```diff
@@ -1,55 +1,83 @@
 # App Bar
 
-The `<pxb-app-bar>` component is a wrapper around the `<mat-toolbar>` that can be resized as the page is scrolled. It supports three modes: `snap`, `collapsed`, and `expanded`.
+The `<pxb-app-bar>` component is a wrapper around the `<mat-toolbar>` that can be resized as the page is scrolled. It supports three variants: `snap`, `collapsed`, and `expanded`.
+
+<div style="text-align:center; margin-bottom:20px">
+    <img width="100%" style="max-width: 600px" alt="App Bar Three Liner " src="./gifs/appBarThreeLiner.gif">
+</div>
 
 ## Usage
 
-```tsx
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 600px" alt="User Menu Anatomy" src="./images/appBarExpanded.png">
+    <img width="100%" style="max-width: 600px" alt="User Menu Anatomy Selector" src="./images/appBarCollapsed.png">
+</div>
+
+```typescript
+// app.module.ts
+import { AppBarComponent} from '@pxblue/angular-components';
+...
+imports: [
+    CommonModule, MatToolbarModule, SpacerModule, MatIconModule
+],
+...
+```
+
+```html
+// AppBar and Three Liner component resizes between collapsedHeight and expandedHeight as the window is scrolled
+<pxb-app-bar variant='snap'>
+    <pxb-three-liner
+        [title] ='title'
+        [subtitle] ='subtitle'
+        [info] ='info'
+    </pxb-three-liner>
+</pxb-app-bar>
```

Comment:
I would move this example after the three minimal examples below (progressive disclosure). And you might want to adjust the comment - the three liner will not automatically resize...that is the user's responsibility. Maybe just "App Bar with a Three Liner component"

Reply:
moved this to last